python/raf/distributed/op.py

```diff
@@ -151,7 +151,25 @@ def reduce_scatter(x, computation="sum", rank_list=None):
         reduction result of x[rank] over all replicas,
         where rank represents rank number of the current process
     """
-    return sym._reduce_scatter(x, computation, rank_list=rank_list)
+    is_list = isinstance(x, (tuple, list))
+
+    comm = get_communicator()
+    if rank_list:
+        for group in rank_list:
+            if comm.rank in group:
+                size = len(group)
+                break
+        else:
+            size = 1
+    else:
+        size = comm.size
+
+    if not is_list:
```

Comment:
Seems not necessary to have `is_list` as it only used once.

```suggestion
    if not isinstance(x, (tuple, list)):
```